README.md

```diff
@@ -264,10 +264,13 @@ the common question options:
 
 The `freetext` directive accepts a second positional argument after the points.
 It defines the compare method for the model solution.
-A textual input can be compared with the model solution as `int`, `float`, `string`,
-or `unsortedchars` (unsorted character set).
-Another option is `regexp` that takes the correct answer as a regular
+A textual input can be compared with the model solution as `int`, `float`,
+`string`, `subdiff`, `regexp` or `unsortedchars` (unsorted character set).
+The `regexp` compare method takes the correct answer as a regular
 expression and tries to match the submission with it.
+The `subdiff` method works almost like the `string` method, but it can have
+multiple correct answers separated with `|` and it shows difference to the each
+correct answer using pythons difflib.
```

Comment:
Instead of referring to the "pythons difflib" that the reader probably knows nothing about, we should explain what kind of feedback is given. "If the submission is incorrect, the feedback shows its difference to the correct solutions. For example, ..."